_posts/2019-01-29-OAS-3.md

```diff
@@ -0,0 +1,91 @@
+---
+title: OpenAPI 3.0
```

Comment:
I'd be more poetic with the title. Something like

```
Introducing experimental support for OpenAPI 3.0
```

if you're @char0n, otherwise perhaps

```
Try out OpenAPI 3.0 in Apiary!
```

if you want to be playful 😉 Up to you, but I'd definitely add more words to make it read like a sentence and be more catchy.